src/main/java/site/coach_coach/coach_coach_server/routine/dto/RoutineListCoachInfoDto.java

```diff
@@ -0,0 +1,13 @@
+package site.coach_coach.coach_coach_server.routine.dto;
+
+import lombok.Builder;
+
+@Builder
+public record RoutineListCoachInfoDto(
+	Long coachId,
+
+	String nickname,
+
+	String profileImageUrl) {
```

Comment:
개행 수정하는 건 어떨까요? 더 가독성 좋을 것 같아서요!
```suggestion
	String profileImageUrl
) {
```

Reply:
넵!

Reply:
아 그리고 여기만 각 변수들 사이에 띄어쓰기가 있던데 .. 모든 dto에 맞추는 것이 좋을 것같아요 
그리고 dto에서는 validate안 하나요??? 🤔